src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,27 +4,28 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
-        // TODO: add log message about method completeOrder was called
+        logger.info("CompleteOrder method was called, Params: userId ={}" + userId);
```

Comment:
The logger info message should use placeholders instead of string concatenation. Change `logger.info("CompleteOrder method was called, Params: userId ={}" + userId);` to `logger.info("CompleteOrder method was called, Params: userId ={}", userId);` to follow best practices for logging.